RecoVertex/BeamSpotProducer/src/PVFitter.cc

```diff
@@ -401,12 +373,12 @@ bool PVFitter::runFitter() {
       upar.Add("x"     , estX       , errX	     , -10.	    , 10.	    ); // 0
       upar.Add("y"     , estY       , errY	     , -10.	    , 10.	    ); // 1
       upar.Add("z"     , estZ       , errZ	     , -30.	    , 30.	    ); // 2
-      upar.Add("ex"    , 0.015	    , 0.01  	     , 0.   	    , 10. 	    ); // 3
-      upar.Add("corrxy", 0.   	    , 0.02  	     , -1.  	    , 1.  	    ); // 4
-      upar.Add("ey"    , 0.015	    , 0.01  	     , 0.   	    , 10. 	    ); // 5
-      upar.Add("dxdz"  , 0.   	    , 0.0002	     , -0.1 	    , 0.1 	    ); // 6
-      upar.Add("dydz"  , 0.   	    , 0.0002	     , -0.1 	    , 0.1 	    ); // 7
-      upar.Add("ez"    , 1.   	    , 0.1   	     , 0.   	    , 30. 	    ); // 8
+      upar.Add("ex"    , 0.010	    , 0.0001  	 , 0.   	, 10. 	    ); // 3
+      upar.Add("corrxy", 0.   	    , 0.02  	 , -1.  	, 1.  	    ); // 4
+      upar.Add("ey"    , 0.010	    , 0.0001  	 , 0.   	, 10. 	    ); // 5
```

Comment:
The PR description says that nothing changes wrt. the previous situation, but here you are changing hard-coded parameter values.
Please clarify the impact of those.